components/table/test/table.vdiff.js

```diff
@@ -664,18 +664,7 @@ describe('table', () => {
 					await expect(elem).to.be.golden();
 				});
 
-				it('col-sort-button-dropdown-open-short-flag-off', async() => {
```

Comment:
We don't need both these tests anymore. The new test captures what we need - the dropdown opening in a short table.